bezierHelper.py

```diff
@@ -1,82 +1,90 @@
-import os, sys
+import os
+import sys
 import pygame
 from pygame.locals import *
 
-if not pygame.font: print("Warining: Fonts Disabled!")
-if not pygame.mixer: print("Warning: Sound Disabled!")
+if not pygame.font:
+    print("Warining: Fonts Disabled!")
+if not pygame.mixer:
+    print("Warning: Sound Disabled!")
+
 
 class BezierHelper():
 
-    #linear interpolate between 2 values
+    # linear interpolate between 2 values
     def Bezier2(self, v1, v2, t, duration=1):
         self.duration = duration
         self.v1 = v1
         self.v2 = v2
-        self.t = self.Clamp01(t/duration)
+        self.t = self.Clamp01(t / duration)
         return v1 + v2 * self.t
 
-    #quadratic interpolate between 3 values
+    # quadratic interpolate between 3 values
     def Bezier3(self, v1, v2, v3, t, duration=1):
         self.duration = duration
         self.v1 = v1
         self.v2 = v2
         self.v3 = v3
-        self.t = self.Clamp01(t/duration)
+        self.t = self.Clamp01(t / duration)
         oneMinusT = 1 - self.t
-        return (oneMinusT * oneMinusT * v1 + \
-        2 * oneMinusT * self.t * v2 + \
-        self.t * self.t * v3)
+        return (oneMinusT * oneMinusT * v1 +
+                2 * oneMinusT * self.t * v2 +
+                self.t * self.t * v3)
```

Comment:
pep8 says the operator should be on the same line as the next operand so in this case `+ 2` is preferable to having the operator on the previous line and it also helps with readability.